src/store/prompts/prompts.epics.ts

```diff
@@ -341,6 +348,11 @@ const publishFolderEpic: AppEpic = (action$, state$) =>
                 : folder.publishVersion,
           }));
 
+        const temporaryFolders = getTemporaryFoldersToPublish(
+          publishedAndTemporaryFolders,
+          newFolders[newFolders.length - 1].folderId,
```

Comment:
Are you sure that correct folder will always be on last place i.e. the order is correct?